internal/verifier/sigstore/container/container.go

```diff
@@ -334,61 +338,75 @@ func getDigestFromVersion(version string) ([]byte, error) {
 }
 
 // bundleFromOCIImage returns a ProtobufBundle based on OCI image reference.
-func bundleFromOCIImage(imageRef string, auth githubAuthenticator) ([]sigstoreBundle, error) {
-	// 1. Get the signature manifest from the OCI image reference
+func bundleFromOCIImage(ctx context.Context,
```

Comment:
If you end up touching the code again, we might want to rename the function to `bundlesFromOCIImage` (plural)

Reply:
ah, you're right, same goes for the other one too 😃